app/Http/Controllers/MatchdayController.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class MatchdayController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        // show ALL matchdays (routes '/' and '/alletermine'; view 'home')
```

Comment:
Nice use of comments to remind your future-self what you wanted to do here. These kinds of comments are usually prefixed with a TODO

// TODO something that needs to be done